index.html

```diff
@@ -3416,17 +3416,17 @@ <h4 id="usage_of_ruby">
               </figure>
             </li>
             <li id="id218">
-              <p its-locale-filter-list="en" lang="en">In the Japanese writing system, <a class="termref" href="#term.compound-word">kanji compound words</a> (jukugo) occasionally appear, usually constructed with a couple of <a class="characterClass" href="#cl-19">ideographic characters (cl-19)</a>. There are two different methods of attaching ruby letters to base <a class="characterClass" href="#cl-19">ideographic characters (cl-19)</a> for these compound words.</p>
+              <p its-locale-filter-list="en" lang="en">In the Japanese writing system, <a class="termref" href="#term.compound-word">kanji compound words</a> (jukugo) occasionally appear, usually constructed with a couple of <a class="characterClass" href="#cl-19">ideographic characters (cl-19)</a>. There are two different methods of attaching ruby characters to base <a class="characterClass" href="#cl-19">ideographic characters (cl-19)</a> for these compound words.</p>
               <p its-locale-filter-list="ja" lang="ja">日本語表記では，複数の<a class="characterClass" href="#cl-19">漢字等（cl-19）</a>で構成された<a href="#term.compound-word" class="termref">熟語</a>をよく使う．この熟語を構成する個々の<a class="characterClass" href="#cl-19">漢字等（cl-19）</a>について，音読み又は訓読みの読み方を示す<a class="characterClass" href="#cl-15">平仮名（cl-15）</a>のルビを付ける（このようなルビの使用例は多い）．このような<span class="index" id="d5e8806"><span class="index" id="d5e8807">熟語に付けるルビ</span></span>について，親文字とルビ文字との対応には，2つの方法がある．</p>
               <ol>
                 <li id="id219">
-                  <p its-locale-filter-list="en" lang="en"><a href="#term.mono-ruby" class="termref2nd">Mono-ruby</a>. Ruby letters are attached to each base <a class="characterClass" href="#cl-19">ideographic character (cl-19)</a>, similarly to the previous section (see [[[#fig2_3_3]]]).</p>
+                  <p its-locale-filter-list="en" lang="en"><a href="#term.mono-ruby" class="termref2nd">Mono-ruby</a>. ruby characters are attached to each base <a class="characterClass" href="#cl-19">ideographic character (cl-19)</a>, similarly to the previous section (see [[[#fig2_3_3]]]).</p>
```

Comment:
capitalize the first character?

Reply:
I need a better tool for editing this... Sorry for the typos!